src/CovidSim.cpp

```diff
@@ -2964,7 +2964,7 @@ int RunModel(int run) //added run number as parameter
 	for (i = j = k = ni = fs2 = i2 = 0; i < P.N; i++)
 	{
 		if (i % 1000 == 0) fprintf(stderr, "\r*** %i              ", i);
-		if (Hosts[i].inf == 0) j++;
+		if (Hosts[i].infectionState == 0) j++;
```

Comment:
Code is commented out.

Either we should uncomment it, or delete it. I'll save that for a future PR.